packages/services/src/terminal/default.ts

```diff
@@ -381,16 +388,10 @@ export class TerminalConnection implements Terminal.ITerminalConnection {
   private _reconnectLimit = 7;
   private _reconnectAttempt = 0;
   private _pendingMessages: Terminal.IMessage[] = [];
+  private _terminalAPIClient: Terminal.ITerminalAPIClient;
 }
 
 namespace Private {
-  /**
-   * Get the url for a terminal.
-   */
-  export function getTermUrl(baseUrl: string, name: string): string {
```

Comment:
Removing this private function as it doesn't seem to be used anywhere in the code base.